src/OLA.php

```diff
@@ -47,7 +47,7 @@ class OLA extends LevelAgreement
 
     public static function getTypeName($nb = 0)
     {
-       // Acronymous, no plural
+        // Acronym, no plural
```

Comment:
While not related to this PR, but as far as I know acronym can have plural form (not in French, but in English or German -- at least).